website/docs/apps/autoplot_app/README.md

```diff
@@ -99,32 +100,34 @@ Tailscale network.
 You can control your Amiga from a web browser at any time, even when the Autoplot app is not active
 on the Brain display.
 
-#### Remote Keyboard Control
+## How to Control your Amiga
 
-Experience direct control of your Amiga robot from any web browser.
-Similar to video game controls, this feature allows you to command the robot using simple keyboard inputs.
+You can control your Amiga robot in two ways:
 
-- **W**: Propels the robot forward
-- **A**: Rotates the robot counter-clockwise on the spot
-- **D**: Rotates the robot clockwise on the spot
+**Manual Control**: Use the [pendant](/docs/pendant),
+the [kartech](https://kar-tech.com/single-axis-joystick-universal-mega-system.html)
+or [dashboard](/docs/dashboard/dashboard-user-guide)
+while physically present with the robot, ideal for teaching the robot new tracks.
 
-These commands engage the robot's autonomy systems to perform the actions safely and precisely,
-setting incremental goals for the robot to achieve.
+Here are instructions for using the pendant and the kartech:
```

Comment:
We really should link this to the actual release notes in amiga-dev-kit, but the release is not yet out, so the direct link will be broken...

```suggestion
Here are instructions for using the pendant and the kartech as of dashboard firmware release [**`v0.5.0`**](https://github.com/farm-ng/amiga-dev-kit/releases):
```

Reply:
Good catch!
Addressed here: 54f6c0888fb3759d41de4f77f032f7e341ef8393